emqx/content.md

```diff
@@ -0,0 +1,299 @@
+# What is EMQ X ?
+
+[EMQ X MQTT broker](https://www.emqx.io/) is a fully open source, highly scalable, highly available distributed MQTT messaging broker for IoT, M2M and Mobile applications that can handle tens of millions of concurrent clients.
+
+Starting from 3.0 release, *EMQ X* broker fully supports MQTT V5.0 protocol specifications and backward compatible with MQTT V3.1 and V3.1.1, as well as other communication protocols such as MQTT-SN, CoAP, LwM2M, WebSocket and STOMP. The 3.0 release of the *EMQ X* broker can scaled to 10+ million concurrent MQTT connections on one cluster.
+
+%%LOGO%%
+
+# How to use this image
+
+### Run emqx
+
+Execute some command under this docker image
+
+`docker run -d --name emqx emqx:$(tag)`
+
+For example
+
+`docker run -d --name emqx -p 18083:18083 -p 1883:1883 emqx:latest`
```

Comment:
````suggestion
```console
$ docker run -d --name emqx -p 18083:18083 -p 1883:1883 %%IMAGE%%:latest
```
````

(Did you want to use something like `:tag` here as well?  Maybe also all the references throughout the docs here?)